examples/enable/basic_draw.py

```diff
@@ -2,8 +2,10 @@
 This demonstrates the most basic drawing capabilities using Enable.  A new
 component is created and added to a container.
 """
-from enable.example_support import DemoFrame, demo_main
-from enable.api import Component, Container, Window
+from traits.api import HasTraits, Instance
+from traitsui.api import Item, View
+
+from enable.api import Component, ComponentEditor
 
 
 class Box(Component):
```

Comment:
The color of the box is defined using rgba values. it would be more readable is we used predefined color constants (which i think exist).

Reply:
Let's leave the contents of the demos for another PR.